sktime/transformers/series_as_features/hog1d.py

```diff
@@ -0,0 +1,195 @@
+import numpy as np
+import pandas as pd
+import numbers
+import math
+from sktime.utils.data_container import tabularize
+from sktime.utils.validation.series_as_features import check_X
+from sktime.transformers.series_as_features.base \
+    import BaseSeriesAsFeaturesTransformer
+
+
+"""
+The HOG1D Transformer proposed by:
+
+@article{zhao2015classifying,
+  title={Classifying time series using local descriptors with hybrid sampling},
+  author={Zhao, Jiaping and Itti, Laurent},
+  journal={IEEE Transactions on Knowledge and Data Engineering},
+  volume={28},
+  number={3},
+  pages={623--637},
+  year={2015},
+  publisher={IEEE}
+}
+"""
+
+
+class HOG1D(BaseSeriesAsFeaturesTransformer):
```

Comment:
Could we also name the new transformers transformers, e.g. DTWTransformer and HOG1DTransformer, makes it easier to recognise for people who aren't very familiar with the methods